src/app/app.server.ts

```diff
@@ -0,0 +1,14 @@
+import express, {Express, Router} from  'express'
+import {router as breadRouter} from "./bread/bread.router";
```

Comment:
Bread entity is not part of this challenge, this router should not be included. You can include routes for other entities like `cake.router.ts`.